src/main/java/ip/Duke.java

```diff
@@ -0,0 +1,53 @@
+package ip;
+
+import ip.command.Command;
+import ip.task.TaskList;
+
+public class Duke {
+    private Storage storage;
+    private TaskList tasks;
+    private Parser parser;
+    private Ui ui;
+
+    public Duke() {
+        storage = new Storage();
+        tasks = storage.readFromFile();
+        parser = new Parser();
+        ui = new Ui();
+    }
+
+    public String getResponse(String input) {
+        Command c = parser.parse(input);
+        return c.execute(tasks, ui);
+    }
+
+    public static class DukeException extends Exception {
+        public DukeException(String msg) {
+            super(msg);
+        }
+    }
+
+//    public static void main(String[] args) {
```

Comment:
According to prof, you shouldn't comment out code and just remove it entirely if you don't need it anymore